command_line.go

```diff
@@ -26,6 +26,7 @@ func ParseCommandLine(app *kingpin.Application) (*cmd.CommandLine, error) {
 	// 2. Parse flags
 	cmdLine := cmd.CommandLine{}
 	app.Flag("config", "path to rsc config file").Short('c').Default(path.Join(os.Getenv("HOME"), ".rsc")).StringVar(&cmdLine.ConfigPath)
+	app.Flag("retry", "Number of retry attempts for non-successful(due to timeouts or server errors) API responses").Short('R').Default("0").IntVar(&cmdLine.Retry)
```

Comment:
Same thing here, `Number of retry attempts for non-successful API responses (500, 503, and timeouts only)` for the description